recipes/viridis/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "viridis" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "59c0598e0e144bda22fefd0174ad827d095f0ef9baf62f51334436e93b7e748b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not py35]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - networkx
+
+test:
+  requires:
+    - pytest
+    - coverage
+    - pytest-cov
+  commands:
+    - py.test
```

Comment:
You're right, I don't have the tests in the tarball currently! Something for the next release. However, I do have the doctests, which in this case provide decent coverage, so I'll try those (which will help me keep the recipe up to date later).